watson-developer-cloud/discovery-components
# chore: add cross-env to scripts requiring env var declaration

#### What do these changes do/fix?
Cross-env allows us to define environment vars in a react script without having those scripts break on Windows. However, it looks like we're only using them for a handful of the scripts that define environment vars in this way. This PR adds `cross-env` to any script that uses the pattern `SOME_ENV_VAR=value run whatever-script`.


#### How do you test/verify these changes?
On a Mac and again on Windows, check that the updated scripts run without needing to be modified.
#### Have you documented your changes (if necessary)?
Shouldn't be necessary, as we already had cross-env partially implemented
#### Are there any breaking changes included in this pull request?
No
